src/middlewares/isBlocked.js

```diff
@@ -0,0 +1,63 @@
+import models from '../api/models';
+import status from '../helpers/constants/status.codes';
+
+const { User } = models;
+/**
+ * check if user is blocked
+ *
+ * @export
+ * @class CheckIfBlocked
+ */
+class CheckIfBlocked {
+  /**
+   * this is a middleware which checks if a user is blocked.
+   *
+   * @static
+   * @param {object} req the request
+   * @param {object} res the response
+   * @param { object } next the next route controller to be called
+   * @memberof CheckUser
+   * @returns {Object} res
+   */
+  static async CheckBlocked(req, res, next) {
```

Comment:
@Ramadhan0 method names should be camelCase. so this function should be `checkBlocked`